Bruin Bite/View Controllers/MenuVC.swift

```diff
@@ -9,10 +9,86 @@
 import UIKit
 import Moya
 import SnapKit
+import Reachability
+
+    class NetworkManager: NSObject {
```

Comment:
Move this class to a new file called `NetworkManager.swift` under the `Helpers` folder.